resources/config_bundles/archlinux/gn_flags.map

```diff
@@ -1,2 +1,3 @@
 custom_toolchain="//build/toolchain/linux/unbundle:default"
 host_toolchain="//build/toolchain/linux/unbundle:default"
+is_official_build=true
```

Comment:
`is_official_build` is responsible for a set of optimization flags. E.g. for 64-bit Linux it enables ThinLTO and control-flow integrity. It slows down the build process but should have advantages during runtime. I experimented with these flags a while back if you remember and 
 recently `is_official_build` made it into the official Arch Linux build script.
I can't comment on the effects for other operating systems though and it will need to be tested.

Additional changes to make it work are listed at [1] and [2]. Specifically the patch for ICU is needed and I believe that the rest is optional.

1: https://git.archlinux.org/svntogit/packages.git/commit/trunk?h=packages/chromium&id=067cef8b936524e5ff1801a57042b2d121ef1363
2: https://git.archlinux.org/svntogit/packages.git/commit/trunk?h=packages/chromium&id=8f15906c7fab24107b34f42f05a4be8e337969e3